examples/PHPExcelAdapter.php

```diff
@@ -21,11 +24,31 @@
 $cell = $worksheet->getCellAt($coordinate);
 $cell->fill('This is cell A1');
 
+// Change the cell's font
+$font = (new Font())->setName('Helvetica');
```

Comment:
This can go to responsibility of aggregation class to change any of default properties which being set on construct (using the Font methods of corse). My idea that all these classes `Font`, `Style`, `Cell` should be aggregated by another class which route them and make sense in global context